src/gateway/wechat-gateway.ts

```diff
@@ -335,7 +335,20 @@ export class WechatGateway extends EventEmitter {
         const rawData: any = []
         let dataLen = 0
 
-        if (response.statusCode !== 200) {
+        if (response.statusCode === 302) {
+          const location = response.headers.location || res.commandUrl
```

Comment:
Let's put some log here for investigation purpose, log out the redirected location here in info level. Also let's add some code here to deal with situation that status code is `301`, which is permanent changed location, log out the location for `301` status code.